CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 #  minimum required cmake version: 3.1.0
 cmake_minimum_required(VERSION 3.1.0)
 
+# This policy provides compatibility with projects that expect the legacy GL library to be used.
+if (POLICY CMP0072 AND UNIX AND NOT APPLE)
```

Comment:
Can you explain why `NOT APPLE`?

Reply:
I understood from the ticket name that I need to fix the issue on U22.
Have we Apple machines to purpose seen new behavior?

Reply:
Looks like you are right,
You can verify on GHA, we have a mac machine
https://github.com/IntelRealSense/librealsense/actions/runs/4183814020/jobs/7248655076
![image](https://user-images.githubusercontent.com/64067618/219968194-acd1541c-b938-4b31-a6b2-61ca081b2b6a.png)

Reply:
Ups, I saw it before)
May I run viewer on Mac and see live that all work correctly?

Reply:
Since you don't have a mac, you can only run MAC with a VM if you want